package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -230,14 +230,31 @@ const FileUploadPreviewWithContext = <
                 audioAttachmentFileContainer,
               ]}
             >
-              <AudioAttachment
-                index={index}
-                item={item}
-                onLoad={onLoad}
-                onPlayPause={onPlayPause}
-                onProgress={onProgress}
+              <View
+                style={[
+                  styles.fileContainer,
+                  index === fileUploads.length - 1
+                    ? {
+                        marginBottom: 0,
+                      }
+                    : {},
+                  {
+                    backgroundColor: white,
+                    borderColor: grey_whisper,
+                    width: -16,
```

Comment:
`width: -16,`

can you think of a comment here to explain why a negative margin.. useful for the future